app/views/my_facilities/bp_controlled.html.erb

```diff
@@ -168,6 +168,30 @@
             </tbody>
           </table>
         <% end %>
+        <div class="d-flex mt-4">
+          <em class="h-12px w-12px mr-8px bg-red br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            0-20%
+          </p>
+        </div>
```

Comment:
There should be a separate grey key for `0%` as grey to match the `.high-is-good` CSS.